app/dashboard/models.py

```diff
@@ -1633,6 +1633,18 @@ def to_dict(self, fields=None, exclude=None):
         return model_to_dict(self, **kwargs)
 
 
+class LabsResearch(models.Model):
+    """Define the structure of Labs Research object."""
+
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=1000)
+    link = models.URLField(null=True)
+    image = models.ImageField(upload_to='labs')
```

Comment:
@aashutoshrathi / @mbeacom should this be made optional?

Reply:
Yes, I think we can do that, since I have added a default image if no image is uploaded.